src/reference/job-script-vars/var-list.txt

```diff
@@ -26,6 +26,9 @@ CYLC_WORKFLOW_HOST                 # Host running the workflow process
 CYLC_WORKFLOW_OWNER                # User ID running the workflow process
 
 CYLC_WORKFLOW_SHARE_DIR            # Workflow (or task!) shared directory (see below)
+CYLC_SHARE_CYCLE_DIR             # Workflow share/cycle/<cycle point directory>
+                                   # for this task. (Use isodatetime to access
+                                   # other cycle points.)
```

Comment:
Not sure it is worth mentioning isodatetime. I don't think this goes into enough detail but I don't think the required amount of detail is appropriate for this reference sheet
```suggestion
CYLC_SHARE_CYCLE_DIR               # Workflow share/cycle/<cycle point directory>
                                   # for this task.
```